crates/swc_bundler/tests/fixture/deno-9591/output/entry.inlined.ts

```diff
@@ -242,7 +242,7 @@ let NATIVE_OS = "linux";
 const navigator = globalThis.navigator;
 if (globalThis.Deno != null) {
     NATIVE_OS = Deno.build.os;
-} else if (navigator?.appVersion?.includes?.("Win") ?? false) {
+} else if ((navigator?.appVersion)?.includes?.("Win") ?? false) {
```

Comment:
In this very specific case `(foo?.bar)?.baz`, you can't actually observe the difference, because the chain is immediately made optional again.